pandas/core/ops/docstrings.py

```diff
@@ -233,6 +233,58 @@ def _make_flex_doc(op_name, typ):
 dtype: float64
 """
 
+_ne_example_SERIES = """
+Examples
+--------
+s1 = pd.Series([1, 2, 3, 4, 5])
+s2 = pd.Series([1, 2, 5, 9, 2])
```

Comment:
This came up in another PR recently but I don't remember if it was yours.  Was there a decision as to whether ">>> " is needed here?